logging/log_storage/logging-config-es-store.adoc

```diff
@@ -1,13 +1,13 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="cluster-logging-log-store"]
-= Configuring the log store
+[id="logging-config-es-store"]
+= Configuring the Elasticsearch log store
 include::_attributes/common-attributes.adoc[]
 include::_attributes/attributes-openshift-dedicated.adoc[]
-:context: cluster-logging-log-store
+:context: logging-config-es-store
 
 toc::[]
 
-{logging-title-uc} uses Elasticsearch 6 (ES) to store and organize the log data.
+You can use Elasticsearch to store and organize the log data.
```

Comment:
I think I would keep the version number in here, to make it visible that the ElasticSearch we use as storage is not the most recent version and thus has limitations compared to an external ES.

Reply:
@jcantrill IMHO stuff like this is a good reason to document the versions of things in the support section of the docs, rather than needing to call it out in individual procedures / topics. Maybe we can reconsider the decision about this? For now I'll re-add it into the doc here.